packages/@sanity/components/src/selects/StyleSelect.js

```diff
@@ -81,58 +85,72 @@ class StyleSelect extends React.Component {
     this.handleOpenList()
   }
 
+  setPopperElement = element => {
+    this._popperElement = element
+  }
+
   render() {
     const {value, transparent, items, className, placeholder, renderItem} = this.props
     const {showList} = this.state
 
     return (
-      <div
-        className={`${styles.root} ${className || ''} ${transparent ? styles.transparent : ''}`}
-        onBlur={this.handleCloseList}
-        onKeyPress={this.handleRootKeyPress}
-        tabIndex={0}
-      >
-        <div className={styles.inner} onClick={this.handleInnerClick}>
-          <div className={styles.selectContainer}>
-            <span className={styles.text}>
-              {value && value.length > 1 && 'Multiple'}
-              {value && value.length == 1 && value[0].title}
-              {!value && placeholder}
-            </span>
-            <div className={styles.arrow}>
-              <FaAngleDown color="inherit" />
+      <Manager>
+        <div
+          className={`${styles.root} ${className || ''} ${transparent ? styles.transparent : ''}`}
+          // onBlur={this.handleCloseList}
+          onKeyPress={this.handleRootKeyPress}
+          tabIndex={0}
+        >
+          <Target>
+            <div className={styles.inner} onClick={this.handleInnerClick}>
+              <div className={styles.selectContainer}>
+                <span className={styles.text}>
+                  {value && value.length > 1 && 'Multiple'}
+                  {value && value.length == 1 && value[0].title}
+                  {!value && placeholder}
+                </span>
+                <div className={styles.arrow}>
+                  <FaAngleDown color="inherit" />
+                </div>
+              </div>
             </div>
-          </div>
+          </Target>
+          <Portal className={styles.portal}>
+            <Escapable onEscape={this.handleCloseList} />
+            {showList && (
+              <Popper placement="bottom-start">
+                <div ref={this.setPopperElement}>
+                  <List className={styles.list}>
+                    {items.map((item, index) => {
+                      const isSemiSelected = value && value.length > 1 && includes(value, item)
```

Comment:
Could you use `value.includes(item)` here instead?

Edit: I see now that this was not added in this PR. Bonus point if you would like to change it to `value.includes(item)` still

Reply:
yep